plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/AgeCommand.java

```diff
@@ -57,64 +59,37 @@ public AgeCommand() {
     // - age <player.location.find_entities.within[20]> adult
     // -->
 
-    private enum AgeType {ADULT, BABY}
+    private enum AgeType { ADULT, BABY }
 
-    @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("entities")
-                    && arg.matchesArgumentList(EntityTag.class)) {
-                scriptEntry.addObject("entities", arg.asType(ListTag.class).filter(EntityTag.class, scriptEntry));
-            }
-            else if (!scriptEntry.hasObject("agetype")
-                    && arg.matchesEnum(AgeType.class)) {
-                scriptEntry.addObject("agetype", AgeType.valueOf(arg.getValue().toUpperCase()));
-            }
-            else if (!scriptEntry.hasObject("age")
-                    && arg.matchesInteger()) {
-                scriptEntry.addObject("age", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("lock")
-                    && arg.matches("lock")) {
-                scriptEntry.addObject("lock", new ElementTag(true));
-            }
-            else {
-                arg.reportUnhandled();
+    public static void autoExecute(ScriptEntry scriptEntry,
+                                   @ArgName("entities") @ArgLinear ObjectTag entities,

```

Comment:
Should be `@ArgSubType(EntityTag.class) List<EntityTag>`

Reply:
oh legacy compensation, I see. I think using SubType with a list will prevent misidentification? ie you don't need order fix if you have explicit types. Not actually sure offhand, it's been a while.

Reply:
I could not get this to work

Reply:
eh alright